src/test/java/org/kiwiproject/dropwizard/error/ErrorContextBuilderTest.java

```diff
@@ -130,6 +146,53 @@ private void assertIllegalArgumentExceptionThrownBuilding(SoftAssertions softly,
         }
     }
 
+    @Nested
+    class CleanupJob {
+
+        @Test
+        void shouldHaveCleanupApplicationErrorsJob() {
+            var cleanupConfig = new CleanupConfig();
+            cleanupConfig.setCleanupJobName("foo-%d");
+
+            var builder = ErrorContextBuilder.newInstance()
+                    .environment(environment)
+                    .serviceDetails(serviceDetails)
+                    .cleanupConfig(cleanupConfig)
+                    .dataStoreType(DataStoreType.NOT_SHARED);
+
+            builder.buildInMemoryH2();
+
+            var dataSourceFactory = ApplicationErrorJdbc.createInMemoryH2Database();
+            builder.buildWithDataStoreFactory(dataSourceFactory);
+
+            var jdbi = Jdbi3Builders.buildManagedJdbi(environment, dataSourceFactory);
+             builder.buildWithJdbi3(jdbi);
+
+             verify(executor, times(3))
```

Comment:
At first I didn't understand why this was three times. I then realized we were using the builder to create three different `ErrorContext`s. This would not be "normal" usage, so would it be better to have separate tests for each of the types we can build? Or is that overkill? _Let's discuss before you  split this test._